src/proxy-animation.js

```diff
@@ -1594,20 +1590,18 @@ export class ProxyAnimation {
   }
 };
 
-function parseTimeRange(value) {
+export function defaultAnimationDelay() { return 'cover 0%'; }
+
+export function defaultAnimationEndDelay() { return 'cover 100%'; }
+
+export function parseTimeRange(value) {
   const timeRange = {
-    start: {
-      name: 'cover',
-      offset: CSS.percent(0)
-    },
-    end: {
-      name: 'cover',
-      offset: CSS.percent(100)
-    }
+    start: { offset: CSS.percent(0) },
```

Comment:
Your change makes it mandatory to have at least one phase name.  As this is not clearly specced, I'm OK with this, but should be aware that this is a change in behavior.  The conditions for the throw should be updated accordingly.

We are still duplicating the start and end offsets.  I realize that this is to cover the case where a phase name is given without offsets.  I'm sure there is a cleaner way of doing this.

Reply:
Updated, I didn't touch parseTimeRange() other than extracting default values in separate functions defaultAnimationDelay() & defaultAnimationEndDelay(). Instead I added another function parseAnimationDelays() that works with 'animation-delay' & 'animation-end-delay'